### PR TITLE
NMS-13515: Reformatting tables

### PR DESCRIPTION
**Pull request by Bonrob2**
### All Contributors

* [ ] Have you read and followed our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [ ] Have you [made an issue in the OpenNMS issue tracker](https://issues.opennms.org)?<br>If so, you should:
  1. update the title of this PR to be of the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the JIRA link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
* [ ] Have you made a comment in that issue which points back to this PR?
* [ ] Have you updated the JIRA link at the bottom of this comment to link to your issue?
* [ ] If this is a new or updated feature, is there documentation for the new behavior?
* [ ] If this is new code, are there unit and/or integration tests?
* [ ] If this PR targets a `foundation-*` branch, does it avoid changing files in `$OPENNMS_HOME/etc/`?

### Pull Request Process

One or more reviewers should be assigned to each PR.

If you know that a particular person is subject matter expert in the area your PR affects, feel free to assign one or more reviewers when you create this PR, otherwise reviewers will be assigned for you.

If you have made additions or changes to the documentation, or if you _need_ documentation for these code changes, please make sure a technical writer has looked it over.

Once the reviewer(s) accept the PR and the branch passes continuous integration, the PR is eligible for merge.

At that time, if you have commit access (are an OpenNMS Group employee or a member of the Order of the Green Polo) you are welcome to merge the PR.
Otherwise, a reviewer can merge it for you.

Thanks for taking time to contribute!

### External References

* JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-13515

### CODE REVIEW

File: docs/modules/operation/pages/service-assurance/monitors/NrpeMonitor.adoc
Comment (mmahacek) [inline docs/modules/operation/pages/service-assurance/monitors/NrpeMonitor.adoc:40]: is this `{\check_name}` or `\{check_name}`?

**Comment by mmahacek** [comment]
I've pushed my review notes directly to this branch.

**Comment by indigo423** [comment]
> I've pushed my review notes directly to this branch.

👍 just fix it immediately in the branch and we do a review with the PR. Thank you!

File: docs/modules/operation/pages/database-reports/database.adoc
Comment (mmahacek) [inline docs/modules/operation/pages/database-reports/database.adoc:233]: Need a check for accuracy on this.  Seems like a duplicated term.  Not sure if correct and unable to tell from the source.

File: docs/modules/operation/pages/operation/daemon-config-files/eventd.adoc
Comment (mmahacek) [inline docs/modules/operation/pages/operation/daemon-config-files/eventd.adoc:31]:
```suggestion
| events/*
```

File: docs/modules/operation/pages/operation/daemon-config-files/eventd.adoc
Comment (mmahacek) [inline docs/modules/operation/pages/operation/daemon-config-files/eventd.adoc:12]: Will conflict with https://github.com/OpenNMS/opennms/pull/3591 from @mfuhrmann

File: docs/modules/operation/pages/operation/newts/configuration.adoc
Comment (mmahacek) [inline docs/modules/operation/pages/operation/newts/configuration.adoc:122]:
```suggestion
.Available caching strategies
```

File: docs/modules/operation/pages/operation/newts/configuration.adoc
Comment (mmahacek) [inline docs/modules/operation/pages/operation/newts/configuration.adoc:123]:
```suggestion
```

File: docs/modules/operation/pages/operation/newts/configuration.adoc
Comment (mmahacek) [inline docs/modules/operation/pages/operation/newts/configuration.adoc:132]:
```suggestion
| org.opennms.netmgt.newts.support.GuavaSearchableResourceMetadataCache
```

File: docs/modules/operation/pages/operation/newts/configuration.adoc
Comment (mmahacek) [inline docs/modules/operation/pages/operation/newts/configuration.adoc:136]:
```suggestion
| org.opennms.netmgt.newts.support.RedisResourceMetadataCache
```

File: docs/modules/operation/pages/operation/newts/configuration.adoc
Comment (mmahacek) [inline docs/modules/operation/pages/operation/newts/configuration.adoc:152]:
```suggestion
| org.opennms.newts.config.cache.redis_hostname
```

File: docs/modules/operation/pages/operation/newts/configuration.adoc
Comment (mmahacek) [inline docs/modules/operation/pages/operation/newts/configuration.adoc:156]:
```suggestion
| org.opennms.newts.config.cache.redis_port
```

File: docs/modules/operation/pages/operation/daemon-config-files/eventd.adoc
Comment (indigo423) [reply docs/modules/operation/pages/operation/daemon-config-files/eventd.adoc:12]: @mfuhrmann @mmahacek I would vote to merge this PR first and adjust PR #3591 afterward accordingly and not worry about the merge conflict here.

**Review by indigo423 — APPROVED** [review]
I think we should squash and clean up the PR to a single commit. I can do this if you want. Just trigger me before we hit the merge button.

**Comment by Bonrob2** [comment]
@indigo423 sounds good. We can merge now or wait until the last two batches are done.

File: docs/modules/operation/pages/database-reports/database.adoc
Comment (Bonrob2) [reply docs/modules/operation/pages/database-reports/database.adoc:233]: Is there some other way to verify this?

**Comment by indigo423** [comment]
> @indigo423 sounds good. We can merge now or wait until the last two batches are done.

Just my opinion: It's an ugly large PR already, we can do the last two batches here before we merge, otherwise we would have two large ugly PR's :D

File: docs/modules/operation/pages/service-assurance/monitors/NrpeMonitor.adoc
Comment (Bonrob2) [reply docs/modules/operation/pages/service-assurance/monitors/NrpeMonitor.adoc:40]: I'm not sure. I didn't change the text from what was originally there.

**Comment by Bonrob2** [comment]
> > @indigo423 sounds good. We can merge now or wait until the last two batches are done.
> 
> Just my opinion: It's an ugly large PR already, we can do the last two batches here before we merge, otherwise we would have two large ugly PR's :D

Yes, the previous time we did a PR for each batch, then merged just the changes from the batch, so it wasn't *quite* as ugly. Let's do one very ugly PR. :-)

File: docs/modules/operation/pages/service-assurance/monitors/NrpeMonitor.adoc
Comment (indigo423) [reply docs/modules/operation/pages/service-assurance/monitors/NrpeMonitor.adoc:40]: Yes, you found a bug and your suggestion is correct. The `{check_name}` is a placeholder name for the NRPE check script running from the NRPE agent on the remote system, so `\{check_name}` is the right thing here.

File: docs/modules/operation/pages/database-reports/database.adoc
Comment (indigo423) [reply docs/modules/operation/pages/database-reports/database.adoc:233]: Agree, it is appended and tracked it down to this here: 
Screenshot 2021-08-30 at 10 54 11

File: docs/modules/operation/pages/service-assurance/monitors/NrpeMonitor.adoc
Comment (Bonrob2) [reply docs/modules/operation/pages/service-assurance/monitors/NrpeMonitor.adoc:40]: Thanks for verifying this!

File: docs/modules/operation/pages/database-reports/database.adoc
Comment (Bonrob2) [reply docs/modules/operation/pages/database-reports/database.adoc]: Thanks for tracking this down!

**Comment by Bonrob2** [comment]
@indigo423 , it looks like we are good to go.

**Review by mmahacek — APPROVED** [review]

**Pull request merged**
